x-pack/test/api_integration/deployment_agnostic/apis/observability/apm/historical_data/has_data.spec.ts

```diff
@@ -8,15 +8,15 @@
 import expect from '@kbn/expect';
 import { apm, timerange } from '@kbn/apm-synthtrace-client';
 import moment from 'moment';
-import { FtrProviderContext } from '../../common/ftr_provider_context';
+import { ApmSynthtraceEsClient } from '@kbn/apm-synthtrace';
+import { DeploymentAgnosticFtrProviderContext } from '../../../../ftr_provider_context';
 
-export default function ApiTest({ getService }: FtrProviderContext) {
-  const registry = getService('registry');
-  const apmApiClient = getService('apmApiClient');
-  const apmSynthtraceEsClient = getService('apmSynthtraceEsClient');
+export default function ApiTest({ getService }: DeploymentAgnosticFtrProviderContext) {
+  const apmApiClient = getService('apmApi');
+  const synthtrace = getService('synthtrace');
 
   // FLAKY: https://github.com/elastic/kibana/issues/177385
```

Comment:
It would be cool to check if it's still [flaky](https://ci-stats.kibana.dev/trigger_flaky_test_runner/1) and remove the comment if possible

Reply:
I have the flaky-test-runner working for some time now https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/7383 to see if it can be removed 👌🏻